src/FireElectroPokemon.java

```diff
@@ -0,0 +1,47 @@
+package src;
+
+public class FireElectroPokemon extends Pokemon{
+    private FirePokemon firePokemonCapabilities;
```

Comment:
I like your creativity, but this won't work. What you would want to do is extend from FirePokemon and from ElectricPokemon, but you can only inherit from one superclass. 
The way it is now, you won't be able to call `boltflame.flameThrowe()` in Main, because flamethower is a method from FirePokemon and `firePokemonCapabilities` is a private attribute.
In the lesson from last week, we learned about interfaces. This case would be perfect for an interface.